frontend/src/components/ConsentPage.js

```diff
@@ -18,10 +18,15 @@ const ConsentPage = (props) => (
       </p>
     </div>
     <div className={styles.consent_page_agreement_btns}>
-      <button type="button">I AGREE</button>
-      <button type="button">NO, TAKE ME AWAY</button>
+      <button onClick={() => pageChange("question")} type="button">
+        {" "}
+        I AGREE
+      </button>
+      <button onClick={() => pageChange("main")} type="button">
+        {" "}
```

Comment:
Just saying, {" "}, this is added by prettier, that automatically format code.
It does not really matter, but you can remove this line of code.